client/a8c-for-agencies/sections/purchases/licenses/license-preview/use-license-actions.ts

```diff
@@ -0,0 +1,85 @@
+import { useTranslate } from 'i18n-calypso';
+import { useMemo } from 'react';
+import getLicenseState from 'calypso/jetpack-cloud/sections/partner-portal/lib/get-license-state';
+import {
+	LicenseState,
+	LicenseAction,
+	LicenseType,
+} from 'calypso/jetpack-cloud/sections/partner-portal/types';
+import { urlToSlug } from 'calypso/lib/url/http-utils';
+import { useDispatch } from 'calypso/state';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+
+export default function useLicenseActions(
```

Comment:
I decided to copy/paste these as it is likely we will have different menu items for Atomic sites in the scope of A4A.